### DISCUSSION TITLE
When will new paramformer large model opensource?

### DISCUSSION THREAD

**Discussion by MonolithFoundation** [modelscope/FunASR#2310] [Q&A] [answered]
The current paraformer is good, but not good enough, any plan to opensource upgrade model?

**Comment by LauraGPT** [comment, marked as answer]
Short answer: there is still no public release date or committed ETA for a successor specifically named a new **Paraformer-Large** checkpoint.

As of 2026-07-16, the current FunASR model table still lists the public offline and streaming Paraformer models as the 220M `paraformer-zh` family. The releases published after this question was opened do not announce a successor Paraformer checkpoint, so it would be misleading to promise a date that has not been published. [Current model table](https://github.com/modelscope/FunASR/blob/1d8080af99fad370645eec7de9d1bcc5ae4ab587/README.md#L140-L156) [Releases](https://github.com/modelscope/FunASR/releases)

What has changed since this question was opened is the recommended model choice:

- Keep **[Paraformer-zh](https://huggingface.co/funasr/paraformer-zh)** when you need the mature non-autoregressive Chinese/English path, character timestamps, hotwords, or the streaming Paraformer contract.
- On a GPU, start an accuracy comparison with **[Fun-ASR-Nano-2512](https://huggingface.co/FunAudioLLM/Fun-ASR-Nano-2512)** for difficult Chinese/English/Japanese audio, Chinese dialects/accents, context, and proper nouns.
- Use the separate **[Fun-ASR-MLT-Nano-2512](https://huggingface.co/FunAudioLLM/Fun-ASR-MLT-Nano-2512)** checkpoint when the requirement is the published 31-language scope.
- On CPU, or when emotion/audio-event output is part of the requirement, compare **[SenseVoiceSmall](https://huggingface.co/FunAudioLLM/SenseVoiceSmall)**.

The Nano checkpoints are not drop-in replacements for every Paraformer output contract. In particular, do not migrate only because the model is newer if your application depends on Paraformer timestamps, hotwords, streaming behavior, latency, or memory usage. The current selection guide explicitly keeps Paraformer for Mandarin timestamp/hotword workloads and recommends benchmarking the alternatives on representative audio. [Selection guide](https://github.com/modelscope/FunASR/blob/1d8080af99fad370645eec7de9d1bcc5ae4ab587/docs/model_selection.md#L5-L43) [Benchmark checklist](https://github.com/modelscope/FunASR/blob/1d8080af99fad370645eec7de9d1bcc5ae4ab587/docs/model_selection.md#L80-L99)

Minimal GPU comparison:

```python
from funasr import AutoModel

# Current Paraformer production contract
paraformer = AutoModel(
    model="paraformer-zh",
    vad_model="fsmn-vad",
    punc_model="ct-punc",
    device="cuda:0",
)

# Accuracy/context candidate for zh/en/ja + Chinese dialects/accents
nano = AutoModel(
    model="FunAudioLLM/Fun-ASR-Nano-2512",
    device="cuda:0",
)

# Separate multilingual checkpoint
mlt_nano = AutoModel(
    model="FunAudioLLM/Fun-ASR-MLT-Nano-2512",
    device="cuda:0",
)
```

Use the same 20-50 representative files and record CER/WER, latency, throughput, peak memory, model revision, and required output fields. That gives an actionable upgrade decision today; a future Paraformer release should only be treated as available after it appears in the official model table or release notes.
